app/views/projects/show.html.erb

```diff
@@ -269,175 +180,289 @@
     </div>
   <% end %>
 
-  <div class="mt-6 bg-white shadow overflow-hidden sm:rounded-lg">
-    <div class="px-4 py-5 sm:p-0">
-      <dl>
-        <% if @project.short_description.present?  %>
-          <div class="sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6 sm:pt-5 sm:pb-2">
-            <dt class="text-sm leading-5 font-medium text-gray-500">
-              Short description
-            </dt>
-            <dd class="mt-1 text-sm leading-5 text-gray-900 sm:mt-0 sm:col-span-2">
-              <%= simple_format Rinku.auto_link(sanitize(@project.short_description), :all, 'class="text-indigo-500" rel="nofollow noopener noreferrer"'), { class: 'mb-3' }, sanitize: false %>
-            </dd>
+  <div class="mt-6 mx-4 gap-0 rounded-lg bg-white overflow-hidden sm:mx-0 sm:grid sm:grid-cols-2">
+    <div class="sm:col-span-1 relative h-40 sm:h-auto bg-cover bg-center" style="background-image: url(<%= @project.image.present? ? url_for(@project.image.variant(resize_to_limit: [1000, 1000])) : '/images/project-default.png' %>);">
+      <% if @project.status.present? %>
```

Comment:
This whole block should be a helper. Also you could use a `case` statement where you just assign variables such as `label`, `color`, and `icon` and just render a partial that takes those values.